examples/glamorous-jest-react/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`enzyme.mount 1`] = `
+.css-1otybxc,
+[data-css-1otybxc] {
+  padding: 4em;
+  background: papayawhip;
+}
+
+.css-1tnuino,
+[data-css-1tnuino] {
+  font-size: 1.5em;
+  text-align: center;
+  color: palevioletred;
+}
+
+<Wrapper>
+  <glamorous(section)>
+    <section
+      className="css-1otybxc"
+    >
+      <Title>
+        <glamorous(h1)>
+          <h1
+            className="css-1tnuino"
+          >
+            Hello World, this is my first glamor styled component!
+          </h1>
+        </glamorous(h1)>
+      </Title>
+    </section>
+  </glamorous(section)>
+</Wrapper>
+`;
+
+exports[`enzyme.render 1`] = `
+.css-1otybxc,
+[data-css-1otybxc] {
+  padding: 4em;
+  background: papayawhip;
+}
+
+.css-1tnuino,
+[data-css-1tnuino] {
+  font-size: 1.5em;
+  text-align: center;
+  color: palevioletred;
+}
+
+<section
+  class="css-1otybxc"
+>
+  <h1
+    class="css-1tnuino"
+  >
+    Hello World, this is my first glamor styled component!
+  </h1>
+</section>
+`;
+
+exports[`enzyme.shallow 1`] = `
+<glamorous(section)>
```

Comment:
Hmmm... I'm not sure why we're not getting the CSS output for the shallow render 🤔 could you look into it a little?